### PR TITLE
Fixed storage tampering issue. Updated UserGuide.

### PR DESCRIPTION
**Pull request by redders7**

### CODE REVIEW

**Comment by redders7** [comment]
Fixes #89

File: text-ui-test/EXPECTED-UNIX.TXT [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Welcome to your Movie Reviews bot,
+Welcome to your myReviews, your personal Movie Reviews bot!
```
Comment (matthewphua) [inline text-ui-test/EXPECTED-UNIX.TXT:1]: nice to update this

File: src/main/java/seedu/duke/Storage.java [hunk 2]
```diff
@@ -88,8 +88,13 @@ public void loadMedia(File f, ReviewList reviewList) {
             fileScanner.close();
 
         } catch (FileNotFoundException e) {
-            System.out.println("error accessing file");
-        } 
+            System.out.println("Error accessing file.");
+        } catch (Exception e) {
+            System.out.println("Sorry, your saved file was corrupted. You may continue using myReviews with a new "
```
Comment (matthewphua) [inline src/main/java/seedu/duke/Storage.java:93]: nice

Comment (matthewphua) [reply src/main/java/seedu/duke/Storage.java:93]: Make sure that this covers all cases tho

**Review by matthewphua — APPROVED** [review]

**Comment by redders7** [comment]
Fixes #149

**Pull request merged**